Trustroots/trustroots
# Migrate list of tribes to React

#### Proposed Changes

Migrate to React the following:

![image](https://user-images.githubusercontent.com/7449720/70574816-476b7580-1ba5-11ea-9324-f00737b39fd4.png)

The [angulargrid](https://yarnpkg.com/en/package/angulargrid) was replaced with custom css, using [styled-components](https://yarnpkg.com/en/package/styled-components).

#### Testing Instructions

* See that the list of tribes behaves as expected.
  Especially check that the grid behaves more or less as expected.

Work on #1133 
~Depends on #1134 and #1135. It's really just the last commit. The dependencies should be merged to `tribes-react` branch before reviewing of this starts.~ (done)
